### DISCUSSION TITLE
Rule contains unsupported option(s): dotprefix

### DISCUSSION THREAD

**Discussion by muskan399** [google/gonids#190] [General]
While parsing a suricata rule with, `..  endswith; http.host; dotprefix;  ..`, getting rule contains unsupported option(s): dotprefix as error

**Comment by duanehoward** [comment]
dotprefix must be a newer keyword to Suricata, that I haven't added. You can file an issue, and pull requests are welcome, but I've not had much time to spend on this lately.
